sql/table.sql

```diff
@@ -2253,3 +2253,6 @@ INSERT INTO meta (species_id, meta_key, meta_value) VALUES (NULL, 'schema_type',
 # Patch identifier
 INSERT INTO meta (species_id, meta_key, meta_value)
   VALUES (NULL, 'patch', 'patch_99_100_a.sql|schema_version');
+INSERT INTO meta (species_id, meta_key, meta_value)
+  VALUES (NULL, 'patch', 'patch_99_100_b.sql|homology_id_64-bits');
+  
```

Comment:
It's the warning here. It means that the line 2258 does not follow the convention of ending with a newline itself. GitHub doesn't show it well but the line only has two whitespace characters and no newline. So in fact, the two whitespace should be removed.